app/cypress/e2e/cif/project-revision/undo-changes.cy.js

```diff
@@ -232,6 +232,32 @@ describe("when undoing, the project revision page", () => {
       "Awaiting Approval"
     );
 
+    // undo quarterly reports
+    cy.findByText(/Quarterly reports/i).click();
+    cy.findByText(/Edit quarterly reports/i).click();
+    cy.findByText(/Quarterly report 1/i).click();
+    cy.get('[aria-label="General Comments"]').clear().type("I will be undone");
+    cy.contains("Changes saved.");
+    cy.findByRole("button", { name: /undo changes/i }).click();
+    cy.findByLabelText(/General Comments/i).should(
+      "have.text",
+      "quarterly report comments 1"
+    );
+
+    //TODO: undo TEIMP agreement once fixture added
```

Comment:
TODOs in code tend to get lost or live forever. Can this missing test be added to an existing card in zenhub as part of the acceptance criteria?

Reply:
Thanks for bringing this up. That part of the code was actually redundant and I added that by mistake but will make a tech debt card for that TODO.

Reply:
TODO card: https://app.zenhub.com/workspaces/climate-action-secretariat-60ca4121764d710011481ca2/issues/gh/bcgov/cas-cif/1568